journey-map/public/style.css

```diff
@@ -189,3 +189,11 @@ form fieldset p {
     stroke-width: 3px;
     filter: drop-shadow(1px 0 0 #ffdd00) drop-shadow(-1px 0 0 #ffdd00) drop-shadow(0 1px 0 #ffdd00) drop-shadow(0 -1px 0 #ffdd00);
 }
+#header-actions {
+    position: fixed;
+    right: 30px;
+}
+#journey-select {
+    margin-right: 30px;
+    margin-left: 3px;
+}
```

Comment:
At the moment the select and search input are slightly different heights... It's not super important, but it'd be nice to make them the same..! (Few options for doing this, easiest might be to make `header-actions` `display: flex` and add `height: 100%`)